index.html

```diff
@@ -12,5 +13,74 @@
 
 <body>
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<div class="container">
+  <header>
+    <div class="nav-container">
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a href="#"><img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo."></a>
+      </nav>
+    </div>
+  </header>
+
+  <div class="top-container">
+    <div class="title-card item">
+      <h1> Innovation<br/> On<br/>Demand<br></h1>
+      <div class="button-div">
+        <button>Get Started</button>
+      </div>  
+    </div>
+
+    <div class="circle-img"> <img src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </div>
+  <div class="main-content-container1">
+    <div class="top-p-container">
+      <div class="top-p">
+        <h3> Features </h3>
+        <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div class="top-p">
+        <h3> About </h3>
+        <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </div>
+  <div>
+    <img class="banner" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  </div>
+  <div class="bottom-p-container">
+    <div class="lower-p">
+      <h3> Services </h3>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    <div class="lower-p">
+      <h3> Product </h3>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    <div class="lower-p">
+      <h3> Vision </h3>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    </div>
+</div>
```

Comment:
```suggestion
</main>
```
It's hard to tell on this interface, I'm trying to suggest a closing main tag for the opening main tag I suggested above, I think this is the right place